src/server/pingserver-rs/src/worker.rs

```diff
@@ -0,0 +1,181 @@
+use crate::session::*;
+use crate::*;
+use std::sync::Arc;
+
+/// A `Worker` handles events on `Session`s
+pub struct Worker {
+    config: Arc<PingserverConfig>,
+    sessions: Slab<Session>,
+    poll: Poll,
+    receiver: Receiver<Session>,
+}
+
+impl Worker {
+    /// Create a new `Worker` which will get new `Session`s from the MPMC queue
```

Comment:
MPSC queue for now?

I am curious, once this is in good shape, to test out the scalability of multithreaded listener, multithreaded worker, and both.

Reply:
That was a typo, using the MPSC queue from the std library